modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -4856,6 +4856,24 @@ Transaction manager's synchronization timeout. Maximum time to wait for internal
 
 ---
 
+=== tombstone_retention_ms
+
+The retention time for tombstone records in a compacted topic. Cannot be enabled at the same time as any of `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, or `cloud_storage_enable_remote_write`.
+
+*Unit:* milliseconds
+
+*Requires restart:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `null`
```

Comment:
Not sure if we usually add recommended values for properties that default to `null`, but up to your discretion as to whether we want to mention a typical default of `24 hours`/`86400000 ms` (see [kafka docs here](https://docs.confluent.io/platform/current/installation/configuration/topic-configs.html#delete-retention-ms)).

Reply:
@WillemKauf per the code, look like this accepts negative values. Should this be a bounded_property to avoid this? I can add 0 as starting point too. 

Asking because it's not always unintended to have negative values in time-based properties.

Reply:
The valid range is actually defined [here](https://github.com/redpanda-data/redpanda/blob/497b802a327739a9b3e430b91d374d1a54b02024/src/v/config/validators.cc#L276-L280): `[1ms, 17592186044415]` is correct.

Reply:
I notice some other time-based properties that might need fixing:

https://docs.redpanda.com/current/reference/properties/cluster-properties/#raft_replica_max_flush_delay_ms is also bounded by `[1ms, 17592186044415]` but is negative in the docs.

https://docs.redpanda.com/current/reference/properties/cluster-properties/#raft_heartbeat_interval_ms is also bounded by `[1ms, 17592186044415]` but is negative in the docs.

There are a lot more properties in core that should have these bounds enforced, but don't... hm.